security.rst

```diff
@@ -474,6 +474,12 @@ You can also manually hash a password by running:
 Read more about all available hashers and password migration in
 :doc:`security/passwords`.
 
+.. versionadded:: 6.2
+
+    In applications using Symfony 6.2 and PHP 8.2 or newer, the
+    `SensitiveParameter PHP attribute`_ is applied to all plain passwords and
+    sensitive tokens so they don't appear in log messages or stack traces.
```

Comment:
```suggestion
    sensitive tokens so they don't appear in stack traces.
```

Re-reading the code PR I guess we need to reword this a bit like this.

Reply:
You are right. Fixed. Thanks.